docs/details/integration/renderers/vg_lite.rst

```diff
@@ -1,8 +1,8 @@
 .. _vglite:
```

Comment:
`.. _vg_lite:`

Reply:
> `.. _vg_lite:`

Done!  This pulled in 1 additional file since that hyperlink anchor name was used in `./docs/details/debugging/vg_lite_tvg.rst`, and it needed to change in both files, of course.